packages/pigeon/README.md

```diff
@@ -228,4 +228,4 @@ abstract class Api2Host {
 ## Feedback
 
 File an issue in [flutter/flutter](https://github.com/flutter/flutter) with the
-word 'pigeon' clearly in the title and cc **@gaaclarke**.
+'pigeon' tag.
```

Comment:
Actually, we should change back to "'pigeon' in the title"; only team members can tag issues.

Reply:
done